meetings/2020_11_24/README.md

```diff
@@ -0,0 +1,110 @@
+# ROS 2 Security Working Group Meeting Minutes
+24 Nov 2020
+
+[Meeting Recording](https://youtu.be/7ZJidRtTqXI) | [Meeting Announcement](https://discourse.ros.org/t/security-wg-meeting/17519)
+
+
+## Agenda
+
+- Administrivia: future meeting minutes
+- G-turtle goals
+  - MoveIt2 security use case
+  - ROS 2 without a file system, [rcl #545](https://github.com/ros2/rcl/issues/545) and [discourse post](https://discourse.ros.org/t/ros-2-without-a-file-system/16942)
+  - [Galactic Roadmap](https://index.ros.org/doc/ros2/Roadmap/#id2) 
+  - sros2 quality status: any comments?
+- ROS2 secure launch and access control
+- [RMF](https://osrf.github.io/ros2multirobotbook) as a use case, see the [demo](https://github.com/osrf/rmf_demos)
+- Revoking keys
+
+## Attendees
+[Iker Luengo Gil](https://github.com/IkerLuengo),
+[Jacob Hassold](https://github.com/jhdcs),
+[Jaime Martin Losa](https://github.com/JaimeMartin),
+[Jeremie Deray](https://github.com/artivis),
+[Kyle Fazzari](https://github.com/kyrofa),
+[Marco Gutierrez](https://github.com/marcoag),
+[Mikael Arguedas](https://github.com/mikaelarguedas),
+[Ruffin White](https://github.com/ruffsl),
+[Sid Faber](https://github.com/sidfaber)
+
+
+## Administrivia
+
+Following a brief discussion, it was decided to move new meeting minutes to the [ros-security/github repo](https://github.com/ros-security/community). Existing meeting minutes in the [ROS wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security) will not be ported.
```

Comment:
```suggestion
Following a brief discussion, it was decided to move new meeting minutes to the [`ros-security/community` Github repository](https://github.com/ros-security/community). Existing meeting minutes in the [ROS wiki](http://wiki.ros.org/ROS2/WorkingGroups/Security) will not be ported.
```

Reply:
Updated in a6e2e0e927a898d850106dbe5e7725534954701b